packages/types/src/fulu/sszTypes.ts

```diff
@@ -31,13 +32,17 @@ export const Cell = new ByteVectorType(BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_
 export const DataColumn = new ListCompositeType(Cell, MAX_BLOB_COMMITMENTS_PER_BLOCK);
 export const ExtendedMatrix = new ListCompositeType(Cell, MAX_BLOB_COMMITMENTS_PER_BLOCK * NUMBER_OF_COLUMNS);
 export const KzgCommitmentsInclusionProof = new VectorCompositeType(Bytes32, KZG_COMMITMENTS_INCLUSION_PROOF_DEPTH);
+export const KZGProofs = new ListCompositeType(
+  denebSsz.KZGProof,
+  FIELD_ELEMENTS_PER_EXT_BLOB * MAX_BLOB_COMMITMENTS_PER_BLOCK
+);
 
 export const DataColumnSidecar = new ContainerType(
   {
     index: ColumnIndex,
     column: DataColumn,
     kzgCommitments: denebSsz.BlobKzgCommitments,
-    kzgProofs: denebSsz.KZGProofs,
+    kzgProofs: KZGProofs,
```

Comment:
DataColumnSidecar should still be `ListCompositeType(denebSsz.KZGProof, MAX_BLOB_COMMITMENTS_PER_BLOCK)` https://github.com/ethereum/consensus-specs/blob/dev/specs/fulu/das-core.md#datacolumnsidecar

```suggestion
    kzgProofs: denebSsz.KZGProofs,
```